R/example_dictionary.R

```diff
@@ -96,7 +96,7 @@ example_id <- function() {
            18L,        4L,        2L,
            13L,        1L,        2L,
             1L,        1L,        3L,
-           16L,        1L,        3L,
+           17L,        1L,        3L,
```

Comment:
Thanks. This seemed to belong to a separate issue/PR. Now `example_products()` has `tilt_subsector` (#803).